.github/workflows/impress.yml

```
@@ -120,29 +106,49 @@ jobs:
       DB_NAME: impress
       DB_USER: dinum
       DB_PASSWORD: pass
-      DB_PORT: 5432
+      DB_PORT: 15432
+      STORAGES_STATICFILES_BACKEND: django.contrib.staticfiles.storage.StaticFilesStorage
+      AWS_S3_ENDPOINT_URL: http://localhost:9000
+      AWS_S3_ACCESS_KEY_ID: impress
+      AWS_S3_SECRET_ACCESS_KEY: password
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
-      - name: Create writable /data
+        uses: actions/checkout@v4
+
+      - name: Setup base
+        run: |
+          make data/media
+          make create-env-files
+
+      - name: Run containers
         run: |
-          sudo mkdir -p /data/media && \
-          sudo mkdir -p /data/static
+          DOCKER_USER=$(id -u):$(id -g)
+          export DOCKER_USER
+          docker compose up --force-recreate -d postgresql
+          docker compose up --force-recreate -d createbuckets
+
       - name: Install Python
         uses: actions/setup-python@v3
         with:
           python-version: '3.10'
+
       - name: Install development dependencies
+        working-directory: src/backend
         run: pip install --user .[dev]
+
       - name: Install gettext (required to compile messages)
         run: |
           sudo apt-get update
           sudo apt-get install -y gettext
+
       - name: Generate a MO file from strings extracted from the project
+        working-directory: src/backend
         run: python manage.py compilemessages
+
       - name: Run tests
-        run: ~/.local/bin/pytest -n 2
+        working-directory: src/backend
+        run: ~/.local/bin/pytest -k test_api_document_versions
```

@AntoLC You should run all the tests:
```suggestion
        run: ~/.local/bin/pytest
```